meson.build

```diff
@@ -1413,6 +1413,14 @@ if features['ios-gl']
     sources += files('video/out/hwdec/hwdec_ios_gl.m')
 endif
 
+v4l2request = get_option('v4l2request').require(
+    cc.has_header_symbol('libavutil/hwcontext.h', 'AV_HWDEVICE_TYPE_V4L2REQUEST')
+)
```

Comment:
I would prefer to check lavu version once it is merged, instead of doing compile test everytime. It is also easier to track, which version is needed for this feature.

Reply:
Yes. Whether we merged this first or not, we would update this test to a version based one once it's in.